pkgs/by-name/sm/smc-fuzzer/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  swiftPackages,
+  fetchFromGitHub,
+  stdenv,
+}:
+
+# This isn’t just useful for fuzzing; it can also be used to explicitly
```

Comment:
I'm not against it, just to clarify why I wrote this: this comment is intended for fellow nixpkgs maintainers who might come across this package in the future and ask "why do we have this fuzzer in nixpkgs?" Well, "A: because it's actually useful as a stand-alone tool." But that's not how the tool describes itself, that's just my reason for adding it.

If that kind of editorializing is allowed in longDescription I'm happy to move it there.

Reply:
I don't think there's any hard guidelines w.r.t what's allowed or not in `longDescription`, but something akin to this comment would definitely be relevant.

```nix
meta.longDescription = ''
  `smc-fuzzer` isn’t just useful for fuzzing; it can also be used to explicitly
  communicate with the SMC. This makes it useful in system management,
  e.g. querying / controlling the charge status of an Apple laptop.
'';
```

Reply:
I know of no restrictions on longDescription, except common sense.